### DISCUSSION TITLE
課題14についての質問（8/30)

### DISCUSSION THREAD

**Discussion by TakedaAyano** [nsc-kiwamu/traning-takeda#72] [Q&A]
問題１についてです。
関数judgeSosuuVoidを使用すると、素数判定がされ出力されるのですが、
関数judgeSosuuを使用すると、素数判定されずにそのまま順に数字がただ出力されていってしまいます。
この原因は、judgeSosuuに数字を渡すことができていないことと、retNumの値が読み取られていないからでしょうか？

**Comment by nsc-kiwamu** [comment]
judgeSosuuは作りかけの状態なのでうまく動いていないです。
素数だったら引数で受け取った値を返す
素数以外だったらundefindを返す
という処理を実装して下さい
